### ISSUE TITLE
UI: show trigger update button when resource list is hidden

### ISSUE THREAD

**Issue by golergka** [tilt-dev/tilt#6439]
## Describe the Feature You Want

I want a button to trigger update to appear in resource view header when resource list is hidden. Here's a mockup:

![image](https://github.com/user-attachments/assets/223cdc0d-759c-4f1b-8454-db3a171a7b84)

### Current Behavior

I often hide resource list on the left to tile Tilt vertically with other windows. However, this means that every time I want to trigger an update, I have to open the resource list, click the button and close it back.

Here's a Jam with a demonstration: https://jam.dev/c/9da8370d-aac9-4374-9821-255bafd45189

### Why Do You Want This?

I want to click less. I think that "trigger update" is one of the main actions a user can take on a resource, which means it should be right next to "Disable Resource" button.

**Comment by nicks** [comment]
are you aware of the keyboard shortcuts in the help menu? i think hitting the `r` will also work.

**Comment by golergka** [comment]
> are you aware of the keyboard shortcuts in the help menu? i think hitting the `r` key will also work.

No, as a matter of fact I did not. Thanks! However, my point about the button still stands. I'll open another issue with suggestion to make this shortcut more discoverable.

**Comment by golergka** [comment]
Also, I'm afraid this shortcut doesn't work after close the left panel and click on the logs.
